src/gui.c

```
@@ -1163,11 +1165,27 @@ void guiShowUIConfig(void)
+{
+    if (delay <= 0)
+        return 0;
+    if (delay <= 3)
+        return 1;
+    if (delay <= 6)
+        return 2;
+    return 3;
+}
+
 // Interface -> Artwork Settings (cover art display + ART.TAR loading).
 void guiShowArtworkConfig(void)
```

Cppcheck (2.21.0)

[style] The function 'guiShowArtworkConfig' should have static linkage since it is not used outside of its translation unit.

(staticFunction)